src/pages/Home.tsx

```diff
@@ -1,40 +1,63 @@
 import React, { useState, useEffect } from "react";
 import axios from "axios";
-import { Pagination, Paper, Toolbar, Typography, AppBar } from "@mui/material";
+import { Pagination, Paper } from "@mui/material";
 import NFTCard from "../components/card/NFTCard";
+import { StyledCardContainer } from "./styledPages/StyledCardContainer.styled";
+import { NFTGrid } from "../components/card/styledCard/NFTGrid.styled";
 
 const NFTsPerPage = 5;
 
-const Home = () => {
-  const [nfts, setNFTs] = useState<any[]>([]);
+interface NFT {
+  id: string;
+  name: string;
+  description: string;
+  image_url: string;
+  background_color?: string;
+  asset_contract: { chain_identifier: string };
+}
+
+const Home: React.FC = () => {
+  const [nfts, setNFTs] = useState<NFT[]>([]);
   const [loading, setLoading] = useState<boolean>(true);
   const [error, setError] = useState<string | null>(null);
   const [page, setPage] = useState(1);
 
-  const API_KEY = "5402e464f97c44968342dd3057b4a262";
-
-  const fetchNFTs = async () => {
+  const fetchNFTs = async (pageNumber: number) => {
     try {
       const response = await axios.get("https://api.opensea.io/api/v1/assets", {
```

Comment:
why not use v2? i think i mentioned this

Reply:
v2 is not available now, maybe in the past was, i will use different URL with collections